microsoft-azure-sb/0.3.9/modules/ROOT/pages/index.adoc

```diff
@@ -68,7 +68,30 @@ image::microsoft-azure-sb-basic.png[basic]
 |*Shared access key name* | Name of the access key configured in the Azure Service Bus namespace. An access key created at a lower level, such as a topic-level shared key, does not work with this option unless you disable the connectivity test at startup.
 |*Shared access key* | 56-bit primary key used to authorize access to storage resources.
 |===
-+
+
+====== Advanced parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+
+| Operation Timeout a| Number |  Amount of time, in milliseconds, to wait for the operation to execute before throwing an error |  30000 |
```

Comment:
Looks great. This is just a memory thing on my part - I see there are no periods at the end of table text. I'm assuming that's how it is being done. I can't find a ref to it in the style guide.